source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1702,9 +1702,12 @@ def event_alert(self):
 		if self in api.getFocusAncestors():
 			return
 		speech.speakObject(self, reason=controlTypes.OutputReason.FOCUS, priority=speech.Spri.NOW)
+		# Ideally, we wouldn't use getPropertiesBraille directly.

```

Comment:
what would be better? why not do that? it would be good to include this in the comment.

Reply:
it appears that using this directly is common practice, I think this comment can be droppped